### PR TITLE
CVE serialize javascript 2.1.2

### PR DESCRIPTION
**Pull request by ashalam**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -6885,7 +6885,7 @@
       "requires": {
         "@babel/code-frame": "^7.0.0",
         "jest-worker": "^23.2.0",
-        "serialize-javascript": "^1.5.0",
+        "serialize-javascript": "^2.1.1",
```
Comment (tjulien) [inline package-lock.json:6888]: would be good to do an npm install to update package-lock.json too

File: package-lock.json [hunk 2]
```diff
@@ -4906,7 +4900,15 @@
       "integrity": "sha1-MAV0OOrGz3+MR2fzhkjWaX11yQM=",
       "dev": true,
       "requires": {
-        "minimist": "0.2.1"
+        "minimist": "0.0.8"
```
Comment (tjulien) [inline package-lock.json:4903]: did we accidentally downgrade `minimist`?

Comment (ashalam) [reply package-lock.json:4903]: no, i think this is due to my past PR not having done `npm ci`. Once this serialize issue is fixed i can go back and fix minimist if it pops up again.

Comment (tjulien) [reply package-lock.json:4903]: This will downgrade minimist to a vulnerable version, we should revert it.

**Pull request closed without merging**